src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -1662,6 +1662,9 @@ internal AnnotationModel AddAnnotation(string titleText, string text, Guid id)
             annotationModel.ModelBaseRequested += annotationModel_GetModelBase;
             annotationModel.Disposed += (_) => annotationModel.ModelBaseRequested -= annotationModel_GetModelBase;
             AddNewAnnotation(annotationModel);
+
+            Logging.Analytics.TrackEvent(Actions.Create, Categories.GroupOperations, "New Group Created", nodes.Count());
```

Comment:
I feel for each of the action tracked, the combination of analytics event category and action name is enough to describe what is happen on a high level. We have been sticking to that so we may not need another detailed description of what this event is. I would maybe save it for what nodes are included rather than just the node count, what do you think. I think this applies to all the changes in this PR but I would like your opinion. There are also other examples in Dynamo code we can refer to.